<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reportDialog</class>
 <widget class="QDialog" name="reportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>668</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#reportDialog{
 background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #320453, stop:1 #802CBD);
}
QGroupBox#groupBox{
	background-color:#7F00FF;
	border-radius: 35px;
}
QTextEdit#reportEdit{
	background-color: #cecece;	
	border-radius: 20px;
	padding-left: 15px;
	padding-top: 5px;
	color: #3a3a3a;
}
QPushButton#confirmButton{
	border-radius: 25px;
	background-color: #1DB500;
	font-weight: bold;
}
QPushButton#confirmButton:hover{
	background-color:#308420;
}
QPushButton#cancelButton{
	border-radius: 25px;
	background-color: #FF4C4C;
	font-weight: bold;
}
QPushButton#cancelButton:hover{
	background-color: #8C1717;
}



</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>431</width>
     <height>501</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Report Problem</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="reportEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>321</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="placeholderText">
     <string>write here.......</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>390</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>511</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ENSIA Library Management System</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
